### ISSUE TITLE
Ability to view rigging to a mesh from specific bone.

### ISSUE THREAD

**Issue by zrksyd** [ScanMountGoat/ssbh_editor#41]
The ability to see which vertices of a mesh are rigged to a certain bone. Can be greyscale or in a rainbow heatmap.

**Comment by ScanMountGoat** [comment]
I don't think a rainbow gradient would be the right choice here since a rainbow doesn't have a clear transition from dark to light. The different colored hue bands tend to create false patterns and cause confusion. For color ramps, I'll probably end up using a color ramp similar to Blender or Maya. Grayscale is also safe option for intensity values like this.
https://docs.blender.org/manual/en/latest/sculpt_paint/weight_paint/introduction.html#the-weighting-color-code

This will require a number of changes to ssbh_wgpu, so it probably won't be implemented any time soon.

**Comment by zrksyd** [comment]
Yeah, that was the color scheme I was thinking of. Also I know it'll take a lot for that to be implemented, I just wanted to put it on your to-do list since the issues tab was a bit lacking.
